insights/contrib/soscleaner.py

```diff
@@ -78,6 +79,10 @@ def __init__(self, quiet=False):
         self.kw_db = dict() #keyword database
         self.kw_count = 0
 
+        # Doing so here to avoid cyclic imports.
+        from insights.client.constants import InsightsConstants as constants
+        self.ip_obfuscation_blacklist = constants.ip_obfuscation_blacklist
```

Comment:
Couldn't this list be maintained as a static list in `soscleaner.py` directly? Is there any advantage to putting it in `insights/client/constants.py`? That way, we'd avoid this cyclic import, and the tests could avoid having to pull `insights.client` modules into contrib.

Reply:
Although the constants import here is the immediate cause of the error, the real culprit is the flawed module structure.

I chose Constants as the first place to go for hardcoded values. The SOSCleaner is still a natural place for a list of excluded Specs.

Reply:
I think `insights.client.constants` should contain only constants that pertain to `insights.client` and its submodules. In this case, `soscleaner` is under `insights.contrib`. Pulling in `insights.client` only for a constant array seems unnecessary to me. We could store the array in the module that needs it. This avoids the cyclic import (in this case), and it avoids furthering the interdependence of the modules in the project. The benefit here (keeping all "constants" in a file called "constants.py") does not outweigh the risk (increased module interdependence and explicit need to import mid-file just to avoid a cyclic import).